app/config/taxRatesAndBands.scala

```diff
@@ -69,6 +82,25 @@ object TaxRatesAndBands {
 
 }
 
+object TaxRatesAndBands20242025MidYearChange extends TaxRatesAndBands {
```

Comment:
can this extend TaxRatesAndBands20242025 and just override the rate changes?

Reply:
Yeah, I've left it out of my review as in the next ticket we will going via the config route and atleast it'll be easier to just copy the whole config from individual bands here than taking from 2 and accidentally missing something.